.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,114 @@
+name: Bug report
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please carefully read [the documentation](https://github.com/sap-linuxlab/community.sap_install/tree/main/docs#readme) before creating a bug report
+
+  - type: dropdown
+    id: role
+    attributes:
+      label: Role
+      description: For what role is the feature you want to propose meant to be?
+      options:
+        - All
+        - sap_anydb_install_oracle
+        - sap_general_preconfigure
+        - sap_ha_install_anydb_ibmdb2
+        - sap_ha_install_hana_hsr
+        - sap_ha_pacemaker_cluster
+        - sap_hana_install
+        - sap_hana_preconfigure
+        - sap_hostagent
+        - sap_install_media_detect
+        - sap_maintain_etc_hosts
+        - sap_netweaver_preconfigure
+        - sap_storage_setup
+        - sap_swpm
+        - None of them (or I don't know)
+    validations:
+      required: true
+
+  - type: dropdown
+    id: os-family
+    attributes:
+      label: OS
+      description: What operating system are you using to run the role?
+      options:
+        - RHEL (7.x, 8.x, 9.x)
+        - SLES (15 SPx)
+        - None of the above (specify in the description area below)
+      default: 2
+    validations:
+      required: true
+
+  - type: textarea
+    id: python-version
+    attributes:
+      label: Python
+      description: Please provide the python version you are using
+      placeholder: 3.9.7
+    validations:
+      required: true
+
+  - type: textarea
+    id: ansible-version
+    attributes:
+      label: Ansible-core
+      description: Please provide the ansible-core version you are using
+      placeholder: 2.12.0
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Please provide a way for us to be able to reproduce the problem you ran into.
+      placeholder: Reproduction
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of what the bug is.
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+      placeholder: Bug description
+    validations:
+      required: true
+
+  - type: dropdown
+    id: additional
+    attributes:
+      label: Additional information
+      description: Do you intend to submit a pr to solve this bug?
```

Comment:
I suggest to use "PR" or "pull request" here. For also supporting entries by new or inexperienced GitHub users, I prefer "pull request".

Reply:
You are right!